wallet/src/Pos/Wallet/Web/Methods/History.hs

```diff
@@ -76,45 +75,45 @@ getFullWalletHistory cWalId = do
     addHistoryTxs cWalId localHistory
     logDebug "getFullWalletHistory: invoked addHistoryTxs"
 
-    !cHistory <- forM fullHistory (constructCTx cWalId walAddrsDetector diff)
-    logDebug "getFullWalletHistory: formed cTxs"
-    pure (cHistory, fromIntegral $ Map.size cHistory)
+    pure (fullHistory, fromIntegral $ Map.size fullHistory)
 
-getHistory
-    :: MonadWalletWebMode m
+getFilteredHistory
```

Comment:
Given it's not immediately obvious what filtering is happening or why, any chance of adding some documentation here? As a newcomer to the codebase, this would help greatly in understanding what's going on.